openjdk/src/test/java/org/conscrypt/AbstractSessionContextTest.java

```diff
@@ -96,6 +96,28 @@ public void testImplicitRemovalOfOldest() {
         assertSessionContextContents(toArray(c, d), toArray(a, b));
     }
 
+    @Test
+    public void testRemoveIfSingleUse() {
+        NativeSslSession a = new MockSessionBuilder().host("a").singleUse(false).build();
```

Comment:
nit: These should probably be named something other than a and b, had to double-check which one was singleUse and multiUse.

Reply:
Done.